src/test/java/org/mockitousage/debugging/InvocationListenerCallbackTest.java

```diff
@@ -95,4 +87,15 @@ public void reportInvocation(MethodInvocationReport mcr) {
             this.locationOfStubbing = mcr.getLocationOfStubbing();
         }
     }
+
+    static Condition<RememberingListener> notifiedFor(final Object returned, final String location) {
+        return new Condition<RememberingListener>() {
+            public boolean matches(RememberingListener toBeAsserted) {
+                assertThat(toBeAsserted.returnValue).isEqualTo(returned);
+                assertThat(toBeAsserted.invocation).isNotNull();
+                assertThat(toBeAsserted.locationOfStubbing).contains(location);
+                return true;
+            }
+        };
+    }
```

Comment:
Since it's used in multiple tests, I suggest to move the method to `org.mockitoutil.Conditions`.